src/lib/db/api-token-store.ts

```diff
@@ -65,8 +65,8 @@ export class ApiTokenStore implements IApiTokenStore {
     async getAllActive(): Promise<IApiToken[]> {
         const stopTimer = this.timer('getAllActive');
         const rows = await this.db<ITokenTable>(TABLE)
-            .where('expires_at', '>', new Date())
-            .orWhere('expires_at', 'IS', null);
+            .where('expires_at', 'IS', null)
+            .orWhere('expires_at', '>', 'now()');
```

Comment:
ok this is not relevant for this PR, but i see from our hosted offering that we use quite a bit of time fetching active api tokens in the background (> 200ms on 99th percentile). This is an attempt to ease on that (but unsure if it actually helps, as I would expect postgres to do these optimizations automatically). 

Switching the order can't hurt though.